### PR TITLE
Update instructions about releasing TFJob

### PR DESCRIPTION
**Pull request by jlewi**
* Point to the instructions in tf-operator repo for how to cut and build
  a tf-operator release.

* Need to update the extra repos in the E2E test so we checkout tf-operator
  at the tag corresponding to the release.

Related to #215

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/532)

### CODE REVIEW

File: releasing.md [hunk 1]
```diff
@@ -25,64 +25,19 @@ create_context.sh $(kubectl config current-context) kubeflow-releasing
...
-```
-
-[kubeflow/testing#43](https://github.com/kubeflow/testing/issues/43) is tracking setup of IAP to make this easier.
-
-Make sure the Argo workflow completes successfully.
-Check the junit files to make sure there were no actual test failures.
-The junit files will be in `gs://kubeflow-releasing-artifacts`.
+Identify the TFJob [release](https://github.com/kubeflow/tf-operator/releases) you
+want to use
+	* If you need to cut a new TFJob release follow the instructions in
```
Comment (ankushagarwal) [inline releasing.md]: nit: markdown formatting issue here.

**Comment by jlewi** [comment]
@willb Try doing "/lgtm" on this PR to see what I mean about anyone in the org can /lgtm.

**Pull request merged**